### PR TITLE
feat(sdk-core): add utils for signer lnd init

### PR DESCRIPTION
**Pull request by saravanan7mani**
Create watch only init data.
Add ip caveat to macaroon.
Signer macaroon permissions.
Get utxo lib network for coin name.

Ticket: BTC-1468

### CODE REVIEW

File: modules/sdk-core/package.json [hunk 1]
```diff
@@ -71,7 +71,9 @@
     "strip-hex-prefix": "^1.0.0",
     "superagent": "^9.0.1",
     "tweetnacl": "^1.0.3",
-    "uuid": "^8.3.2"
+    "uuid": "^8.3.2",
+    "macaroon": "^3.0.4",
```
Comment (sachushaji) [inline modules/sdk-core/package.json:75]: Seems not a very well maintained library, Last publish is 6 years ago.

Comment (sachushaji) [reply modules/sdk-core/package.json:75]: Any other possible alternatives?

Comment (saravanan7mani) [reply modules/sdk-core/package.json:75]: Unfortunately no. Other alternative libs also has similar maintenance. This has most downloads. Writing macaroon functions are very low level.

BTW, reason for low maintenance- macaroon tokens are not popular and its like JWT, nothing much to improve.

**Review by sachushaji — APPROVED** [review]
package changes lgtm, please get review from btc team for utxo changes

**Pull request merged**